asab/library/service.py

```diff
@@ -272,6 +272,9 @@ async def _list(self, path, tenant, providers):
 				L.exception("Error when listing items from provider", exc_info=ress)
 				continue
 
+			if not ress:
+				continue
```

Comment:
The provider should either do exception or return a list.
`None` as a return value is prohibited, please fix the provider instead.